### DISCUSSION TITLE
error when trying to open a OpenSCAD file

### DISCUSSION THREAD

**Discussion by Gfcima** [BelfrySCAD/BOSL2#1342] [General]
dont know the answer to this error: ERROR: Parser error in file "C:/Users/gcima/OneDrive/Documents/OpenSCAD/libraries/BOSL2/attachments.scad", line 2008: syntax error 

attachments.scad line 2008   bcs = function(list, i=0, inv=false, out=[])
                    i>=len(list)? out :
                    let( nu_inv = list[i]? !inv : inv )
                    bcs(list, i+1, nu_inv, [each out, nu_inv]),
            inverts = bcs([
                false,
                for(i = idx(edge_string)) if (i)
                    _edge_transition_needs_flip(
                        edge_string[i-1],
                        edge_string[i]
                    )
            ]),

**Comment by wdnick** [comment]
I'm getting the same error.  Interestingly enough I downloaded and used this same library on a different computer yesterday with issue.

**Reply by mayih** [reply]
I have the same error...
